doc: known_issue: Matter add known issue to KRKNWK-18916 bug (#15180)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/known_issues.rst

The issues in this section are related to the :ref:`ug_matter` protocol.

> .. rst-class:: v2-5-0 v2-5-1 v2-5-2 v2-5-3 v2-6-0 v2-6-1

Reviewer: ```suggestion
.. rst-class:: v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0
```

> KRKNWK-18916: If the :kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_ERASE_AND_PAIRING_START` is selected there are two issues:

Reviewer: ```suggestion
KRKNWK-18916: If the :kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_ERASE_AND_PAIRING_START` is selected there are two issues:
```
```suggestion
KRKNWK-18916: Issues related to the  :kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_ERASE_AND_PAIRING_START` Kconfig option
  When the Kconfig option is selected, there are two issues:
```
The formatting requires a description line without an empty line, which can't be done with a bullet point.

>   * An assert after removing the last fabric may occurs because OpenThread interface is still active despite of disabling Thread stack.

Reviewer: ```suggestion
  * An assert may occur after removing the last fabric because the OpenThread interface is still active despite the Thread stack being disabled.
```

>   * Cannot commission the device to Matter network because device's MAC address is the same as the saved in SRP server.

Reviewer: ```suggestion
  * The device cannot be commissioned to the Matter network because the device's MAC address is the same as the one saved in the SRP server.
```

  **Workaround:** Add the following lines to the :file:`fabric_table_delegate` file:

>     * After the line 54 (After the ``DoFactoryReset()`` function):

Reviewer: ```suggestion
    * After Line 54 (After the ``DoFactoryReset()`` function):
```

      .. code-block:: C++

        chip::DeviceLayer::ThreadStackMgrImpl().LockThreadStack();
        otIp6SetEnabled(chip::DeviceLayer::ThreadStackMgrImpl().OTInstance(), false);
        chip::DeviceLayer::ThreadStackMgrImpl().UnlockThreadStack();

>     * After the line 56 (After the ``ErasePersistentInfo()`` function):

Reviewer: ```suggestion
    * After Line 56 (After the ``ErasePersistentInfo()`` function):
```

      .. code-block:: C++

        otExtAddress newOtAddr = {};
        chip::Crypto::DRBG_get_bytes(reinterpret_cast<uint8_t*>(&newOtAddr), sizeof(newOtAddr));
        otLinkSetExtendedAddress(chip::DeviceLayer::ThreadStackMgrImpl().OTInstance(), &newOtAddr);

  .. note::

>     for |NCS| v2-5-0 v2-5-1 v2-5-2 revisions the :file:`fabric_table_delegate.h` file is located in the `samples/matter/common/src/` directory, whereas for |NCS| v2-6-0 v2-6-1 revisions the file is located in the `samples/matter/common/src/app` directory.

Reviewer: ```suggestion
    For |NCS| versions v2.5.3, v2.5.2, v2.5.1, and v2.5.0, the :file:`fabric_table_delegate.h` file is located in the :file:`samples/matter/common/src/` directory, whereas for |NCS| versions v2.6.1 and v2.6.0, the file is located in the :file:`samples/matter/common/src/app` directory.
```

The issues in this section are related to the :ref:`ug_matter` protocol.

.. rst-class:: v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0

KRKNWK-18916: Issues related to the  :kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_ERASE_AND_PAIRING_START` Kconfig option
  When the Kconfig option is selected, there are two issues:

  * An assert may occur after removing the last fabric because the OpenThread interface is still active despite the Thread stack being disabled.

>   * The device cannot be commissioned to the Matter network because the device's MAC address is the same as the one saved in the SRP server.

Reviewer 2: I don't think it's about MAC address, right? IEEE 802.15.4 Extended address is not the same as MAC address. In  therms of Thread EUI64 is more something like the MAC address.

Author: Changed.
